core/ir/riscv64/encode.c

```diff
@@ -102,6 +102,7 @@ instr_encode_arch(dcontext_t *dcontext, instr_t *instr, byte *copy_pc, byte *fin
 {
```

Comment:
Re PR description:
nit: Maybe make it a bit more verbose:
```
Fixes instr_encode_arch by adding missing handling of compressed instructions.
```

Reply:
Yes better, done.